content/exchange-guide.md

```diff
@@ -0,0 +1,575 @@
+
+A user named Charlie wants to deposit 50,000 XRP to Alpha Exchange. Doing this involves the following steps:
+
+1. Charlie submits a payment of 50,000  XRP (by using [RippleAPI](https://ripple.com/build/rippleapi/) or similar software) to Alpha Exchange's [issuing account](#accounts).
+
+    a. Charlie adds an identifier (in this case, `789`) to the payment to associate it with his account at Alpha Exchange. This is called a [_destination tag_](https://ripple.com/build/gateway-guide/#source-and-destination-tags). (To use this, Alpha Exchange must have set the asfRequireDest flag on all of its accounts. This flag requires all incoming payments to have a destination tag like Charlie's. For more information, see [AccountSet Flags](https://ripple.com/build/transactions/#accountset-flags).
+
+2. The software at Alpha Exchange detects the incoming payment, and recognizes `789` as the destination tag for Charlie’s account.
+
+3. When it detects the incoming payment, Alpha Exchange's software updates its balance sheet to indicate that the 50,000 XRP it received is controlled by Charlie.
+
+    Charlie can now use up to 50,000 XRP on the exchange. For example, he can create offers to trade XRP with BTC or any of the other currencies Alpha Exchange supports.
+
+<table>
+  <tr>
+    <td><b><i>XRP Balances
+on RCL</i></b></td>
+    <td></td>
+    <td></td>
+    <td><b><i>Alpha Exchange
+XRP Balances</i></b></td>
+    <td></td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><b>User</b></td>
+    <td><b>Balance</b></td>
+    <td></td>
+    <td><b>Acct #</b></td>
+    <td><b>User</b></td>
+    <td><b>Balance</b></td>
+  </tr>
+  <tr>
+    <td>Dave</td>
+    <td>25,000</td>
+    <td></td>
+    <td>123</td>
+    <td>Alice</td>
+    <td>0</td>
+  </tr>
+  <tr>
+    <td>Edward</td>
+    <td>45,000</td>
+    <td></td>
+    <td>456</td>
+    <td>Bob</td>
+    <td>0</td>
+  </tr>
+  <tr>
+    <td>Charlie</td>
+    <td><s>100,000</s>
+<br>50,000</td>
+    <td></td>
+    <td>789</td>
+    <td>Charlie</td>
+    <td><s>0</s>
+<br>50,000</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td></td>
+    <td></td>
+    <td></td>
+    <td></td>
+    <td></td>
+  </tr>
+  <tr>
+    <td>Alpha Operational</td>
+    <td>0</td>
+    <td></td>
+    <td>...</td>
+    <td></td>
+    <td></td>
+  </tr>
+  <tr>
+    <td>Alpha Standby</td>
+    <td>0</td>
+    <td></td>
+    <td></td>
+    <td></td>
+    <td></td>
+  </tr>
+  <tr>
+    <td>Alpha Issuing</td>
+    <td><s>0</s>
+<br>50,000</td>
+    <td></td>
+    <td></td>
+    <td></td>
+    <td></td>
+  </tr>
+  <tr>
+    <td>...</td>
+    <td></td>
+    <td></td>
+    <td></td>
+    <td></td>
+    <td></td>
+  </tr>
+</table>
+
+
+### Trade XRP on The Exchange
```

Comment:
"The" should not be capitalized. But maybe the header should say "Trade XRP on Alpha Exchange" instead.